docs/config.md

```diff
@@ -139,7 +139,8 @@ SECRET_KEY = config('SECRET_KEY', cast=Secret)
 
 DATABASE_URL = config('DATABASE_URL', cast=URL)
 if TESTING:
-    DATABASE_URL = DATABASE_URL.replace(database='test_' + DATABASE_URL.database)
+    db_name = DATABASE_URL.path.lstrip('/')
+    DATABASE_URL = DATABASE_URL.replace(path='/test_' + db_name)
```

Comment:
I guess the real intentions here were those: https://github.com/encode/starlette/pull/774
```suggestion
    DATABASE_URL = DATABASE_URL.replace(database='test_' + DATABASE_URL.database)
```

Reply:
This will be fixed on https://github.com/encode/starlette/pull/774/